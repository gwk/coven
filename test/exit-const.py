def f():
  exit(0)

f() #!cov-ignore.
