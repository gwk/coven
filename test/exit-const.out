----------------
Coverage Report:

__main__: exit-const.py: 3 traceable; 2 covered; 1 ignored; 0 ignored but covered; 0 not covered.
